app/controllers/automation_manager_controller.rb

```diff
@@ -0,0 +1,1025 @@
+    end
+  end
+
+  def self.model_to_type_name(provmodel)
+    if provmodel.include?("ManageIQ::Providers::AnsibleTower")
+      'ansible_tower'
+    end
+  end
+
+  def model_to_name(provmodel)
+    AutomationManagerController.model_to_name(provmodel)
+  end
+
+  def model_to_type_name(provmodel)
+    AutomationManagerController.model_to_type_name(provmodel)
+  end
+
+  def index
+    redirect_to :action => 'explorer'
+  end
+
+  def show_list
+    redirect_to :action => 'explorer', :flash_msg => @flash_array.try(:fetch_path, 0, :message)
+  end
+
+  def new
+    assert_privileges("automation_manager_add_provider")
+    @provider_manager = ManageIQ::Providers::AnsibleTower::AutomationManager.new
+    @server_zones = Zone.in_my_region.order('lower(description)').pluck(:description, :name)
+    render_form
+  end
+
+  def edit
+    @server_zones = Zone.in_my_region.order('lower(description)').pluck(:description, :name)
+    case params[:button]
+    when "cancel"
+      cancel_provider_automation_manager
+    when "save"
+      add_provider_automation_manager
+      save_provider_automation_manager
+    else
+      assert_privileges("automation_manager_edit_provider")
+      manager_id            = from_cid(params[:miq_grid_checks] || params[:id] || find_checked_items[0])
+      @provider_manager     = find_record(ManageIQ::Providers::AnsibleTower::AutomationManager, manager_id)
+      @providerdisplay_type = model_to_name(@provider_manager.type)
+      render_form
+    end
+  end
+
+  def delete
+    assert_privileges("automation_manager_delete_provider")
+    checked_items = find_checked_items
+    checked_items.push(params[:id]) if checked_items.empty? && params[:id]
+    providers = ManageIQ::Providers::AutomationManager.where(:id => checked_items).includes(:provider).collect(&:provider)
+    if providers.empty?
+      add_flash(_("No %{model} were selected for %{task}") % {:model => ui_lookup(:tables => "providers"), :task => "deletion"}, :error)
+    else
+      providers.each do |provider|
+        AuditEvent.success(
+          :event        => "configuration_manager_record_delete_initiated",
+          :message      => "[#{provider.name}] Record delete initiated",
+          :target_id    => provider.id,
+          :target_class => provider.type,
+          :userid       => session[:userid]
+        )
+        provider.destroy_queue
+      end
+
+      add_flash(n_("Delete initiated for %{count} Provider",
+                   "Delete initiated for %{count} Providers",
+                   providers.length) % {:count => providers.length})
+    end
+    replace_right_cell
+  end
+
+  def refresh
+    assert_privileges("automation_manager_refresh_provider")
+    @explorer = true
+    automation_manager_button_operation('refresh_ems', _('Refresh'))
+    replace_right_cell
+  end
+
+  def tagging
+    case x_active_accord
+    when :automation_manager_providers
```

Comment:
The configured systems are displayed on 2 different accordions - each with it's own privilege ( this is the same as in the foreman_provider)